packages/api-plugin-fulfillment-method-shipping-ups/src/util/validateOrderMethodsups.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @summary Sample dummy function validating the data requirements for this metod
+ * @param {Object} context - Context object
+ * @param {Object} commonOrder - Current order which provide available details to perform validation
+ * @param {Object[]} validationResults - Validation results collected till now
+ * @returns {Object[]} validationResults - with the validation details populated
+ */
+export default function validateOrderMethodsups(context, commonOrder, validationResults = []) {
```

Comment:
This is not correctly camel-cased

Reply:
We have to apply the same fix here that we discussed to avoid string concatenation. I shall apply the same after the initial PR merging

Reply:
On second thought - I went ahead and applied this change in this PR. I shall make corresponding changes to other files in respective PRs